media-sound/supercollider/supercollider-3.8.0-r1.ebuild

```diff
@@ -65,7 +63,6 @@ src_configure() {
 	local mycmakeargs=(
 		-DAUDIOAPI=$(usex jack jack portaudio)
 		-DINSTALL_HELP=ON
-		-DNATIVE=ON
```

Comment:
You are changing the install image without making a revbump. I suspect you make the same change in 3.8.1 and 3.9.4, so if .0 was wrong in that regard, simply remove the old ebuild in a separate commit.

Reply:
@a17r Do you think that change is worth a revbump/recompile? I wasn't sure, it was wrong of course, but not sure if it would be worth a recompile.
I can drop the change to the 3.8.0-r1 ebuild as well. What do you prefer?

Reply:
well, the point was that you drop the changes to -r1, and drop the ebuild altogether after those bumps.

Reply:
@a17r Just pushed the changes, hope it's OK like this :)